lib/src/method_call_handler.dart

```diff
@@ -25,4 +27,20 @@ class AblyMethodCallHandler{
     return callbackResponse;
   }
 
+  bool realtimeAuthInProgress = false;
```

Comment:
I think this should be a `Future` that is returned to all calls if not `null` instead of a boolean flag.
This way callers can wait for the callback to complete.

Reply:
@zoechi Caller here is Android/iOS platform which would take no action when `null` is returned. And returning the same future will return same results to platform side which will have undesirable effects on the platform side.